.gitignore

```diff
@@ -6,4 +6,5 @@
 \.gradle
 \build
 AdvancedTeleport-Bukkit\.gradle
-AdvancedTeleport-Bukkit\build
\ No newline at end of file
+AdvancedTeleport-Bukkit\build
+AdvancedTeleport-Bukkit\src\main\resources
```

Comment:
Are you sure that ignoring the resources directory is the right call? That **will** lead to nasty surprises.

I know it is rude but I feel like I shouldn't be updating to 5.7 should it ever release. The churn is immense and the little I do end up reviewing is mind-boggling.

Reply:
Crap, wrong directory, I'll fix that

Is there anything specific about it that you're worried about? I'm starting to settle it into being v6 due to the large changes being made. After that I've pretty much decided I won't do this kind of thing again and only push one major feature at a time.